control/control.leanos.xml

```diff
@@ -111,7 +111,7 @@ textdomain="control"
 
         <!-- bnc#875350: Explicitly selecting these patterns by default -->
         <!-- For a list of patterns, see https://build.suse.de/package/view_file/SUSE:SLE-12-SP3:GA/patterns-sles/patterns-sles.spec?expand=1 -->
-        <default_patterns>minimal_base</default_patterns>
+        <default_patterns>base</default_patterns>
```

Comment:
this is the set of patterns that are always selected by proposal. But any product can overwrite it in its skelcd, but I do not see yet any module doing it, so it use this shared one.
Change in general looks good for me. Just please add entry to changelog(including bzilla or jira entry ) and bump version.